.test-infra/jenkins/job_PerformanceTests_FileBasedIO_IT.groovy

```diff
@@ -28,7 +28,9 @@ def jobs = [
                 pipelineOptions    : [
                         bigQueryDataset: 'beam_performance',
                         bigQueryTable  : 'textioit_results',
-                        numberOfRecords: '1000000'
+                        numberOfRecords: '1000000',
+                        maxNumWorkers  : '5',
```

Comment:
I think a proper set of parameters would be: 

```
numWorkers            : 5,
autoscalingAlgorithm  : "NONE"
```

the first determines how many workers to use, the second explicitly states that there is no autoscaling. I'm not sure whether current parameter set acheves what we want to achieve (disable autoscaling)

Reply:
Other (also important) thing: what is the reasoning behind setting 5 workers everywhere? Maybe we're fine with 1 worker or 2 (to test parallelism as well)? WDYT? What things should we consider when choosing the right amount? 

We never really discussed this so I think this is the right moment and about time. :)

@ibzib @chamikaramj @pabloem what is your opinion about this? Could you share your insights?

Reply:
1. I think it makes sense to deactivate autoscaling, because the behavior would be non-deterministic if we have autoscaling
2. I think the number of workers should be chosen to make the pipeline run relatively quickly, and hopefully be more than one, so we measure any effects from separate workers.

Reply:
So to conclude: I agree with @lgajowy 's suggestion

Reply:
Thanks @pabloem - let's leave 5 workers and deactivate autoscaling then. :)